books.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
```

Comment:
- [ ] Kindly make the `rentals` attribute have a getter and setter method. you can use the `attr_accessor` for it. Remember that the `rentals` will be the link between the `Book` and `Person` classes. 👍